### ISSUE TITLE
Update to new fsspec-xrootd 

### ISSUE THREAD

**Issue by alexander-held** [usatlas/analysisbase-dask#41]
The latest `fsspec-xrootd` release [v0.3.0](https://github.com/CoffeaTeam/fsspec-xrootd/releases/tag/v0.3.0) brings some updates (I think especially https://github.com/CoffeaTeam/fsspec-xrootd/pull/54) that might be quite relevant for IDAP testing so I'd propose to update AB-dev.

**Comment by matthewfeickert** [comment]
Resolved by PR #42.

(Closing manually again as linking isn't working. :?)

**Comment by matthewfeickert** [comment]
On AB-dev:

```console
[bash][feickert]:~ > python -m pip show fsspec-xrootd
Name: fsspec_xrootd
Version: 0.3.0
Summary: xrootd implementation for fsspec
Home-page: https://github.com/CoffeaTeam/fsspec-xrootd
Author: Scott Demarest
Author-email: rockets1194@gmail.com
License: BSD-3-Clause
Location: /venv/lib/python3.9/site-packages
Requires: fsspec
Required-by: coffea
[bash][feickert]:~ > 
```
